docs/program/apis/robot.md

```diff
@@ -7,11 +7,157 @@ description: "How to use the Robot API to monitor and manage your smart machines
 tags: ["robot state", "sdk", "apis", "robot api"]
 ---
 
-The _robot API_ is the application programming interface that manages each of your robots running `viam-server`.
-It is the API for high level operations of each robot {{< glossary_tooltip term_id="part" text="part" >}}.
+The _robot API_ is the application programming interface that manages each of your smart machines running `viam-server`.
+Use the robot API to connect to your smart machine from within a supported [Viam SDK](/program/apis/), and send commands remotely.
+
+The robot API is supported for use with the [Viam Python SDK](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient), the [RDK (the Viam Go SDK)](https://pkg.go.dev/go.viam.com/rdk/robot/client#RobotClient), and the [Viam C++ SDK](https://cpp.viam.dev/classviam_1_1sdk_1_1RobotClient.html).
```

Comment:
```suggestion
The robot API is supported for use with the [Viam Python SDK](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient), the [Viam Go SDK](https://pkg.go.dev/go.viam.com/rdk/robot/client#RobotClient), and the [Viam C++ SDK](https://cpp.viam.dev/classviam_1_1sdk_1_1RobotClient.html).
```
I think RDK is what they all interface with, right, although it's technically in Go? I might be confused on technical def here so up to you

Reply:
Perfect! Thanks, if we can just call it the Go SDK directly, that's super helpful. Thank you!